src/structures/Widget.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const Base = require('./Base');
+const Collection = require('../util/Collection');
+
+/**
+ * Represents a Widget.
+ */
+class Widget extends Base {
+  /**
+   * @param {Client} client - The instantiating client
```

Comment:
Do we do the `-` anywhere else?

Reply:
We do in `@discordjs/voice`, but I don't recall this in the base d.js repository.